src/app/route-info.js

```diff
@@ -13,13 +13,14 @@ export function displayRegionInfo (selection) {
   // create the feature info box
   if (!featureFlag) createFeatureInfo()
   setPosition(selection.pixel.x, selection.pixel.y)
-
-  const { speed, osmlr_id, id } = selection.feature.properties
+  /* eslint-disable camelcase */
+  const { speed, osmlr_id, id } = selection.feature.properties 
```

Comment:
looks like there are trailing spaces introduced here that is now breaking the test